source/load-ivy.sc

```diff
@@ -1,5 +1,11 @@
-interp.repositories() ++= Seq( 
-  coursier.maven.MavenRepository("https://oss.sonatype.org/content/repositories/snapshots")
+// interp.repositories() ++= Seq( 
```

Comment:
What error does this fix? Is it #100?

Reply:
This fix an error w.r.t. coursier api update when running the bootcamp locally, similar to [https://github.com/lihaoyi/Ammonite/issues/1003](url)